libmscore/glissando.cpp

```diff
@@ -220,7 +220,11 @@ void Glissando::layout()
 
       // FINAL SYSTEM-INITIAL NOTE
       // if the last gliss. segment attaches to a system-initial note, some extra width has to be added
-      if (cr2->segment()->measure() == cr2->segment()->system()->firstMeasure() && cr2->rtick() == 0)
```

Comment:
Well, no, actually. Calling the `segment()` of a `Chord` does indeed return a `Segment` (and to all evidence **its** `Segment`), as it can be seen at: https://github.com/musescore/MuseScore/blob/master/libmscore/chord.cpp#L2867